Framework/CurveFitting/src/Algorithms/RefinePowderInstrumentParameters.cpp

```diff
@@ -295,8 +295,7 @@ void RefinePowderInstrumentParameters::fitInstrumentParameters() {
   for (size_t i = 0; i < m_dataWS->readX(0).size(); ++i)
     outss << m_dataWS->readX(0)[i] << "\t\t" << m_dataWS->readY(0)[i] << "\t\t"
           << m_dataWS->readE(0)[i] << '\n';
-  cout << "Input Peak Position Workspace To Fit: " << endl
-       << outss.str() << '\n';
+  cout << "Input Peak Position Workspace To Fit: \n" << outss.str() << '\n';
```

Comment:
Should we get rid of any usages of  `cout`/`cerr` like this? They don't seem like a good idea in the middle of an algorithm.